### ISSUE TITLE
detect and propagate edits on mastodon to bluesky

### ISSUE THREAD

**Issue by j2bryson** [snarfed/bridgy-fed#1643]
I can't believe this is a new issue / feature request, but I can't find it here yet. The main reason I love mastodon is I can actually edit the posts. But everyone is on bluesky. Sometimes I make dumb mistakes and want to edit them. Would it be possible to detect the changes, delete the bluesky post and post the new version as a new post? Even just posting the new version as a new post without the delete would be better than leaving a stupid mistake there – like a photo that disclosed more information than you'd originally noticed.

**Comment by snarfed** [comment]
Definitely the right idea! We've done this since the beginning. Sadly we're still waiting on Bluesky and bsky.app to surface edits visibly to users: #947. It's on their roadmap at least!

**Comment by snarfed** [comment]
Duplicate of #947
